src/layout/rectangles.js

```diff
@@ -49,33 +53,34 @@ export default function(layoutStrategy) {
             // or number of rectangles.
             var layout = strategy(childRects);
 
-            // add indices to the layouts
+            // add indices to the layouts - this is require in order to determine which
```

Comment:
After discussion -
* [ ] add optional `hidden` flag to objects returned from layout
* [ ] remove `filter` property
* [ ] consolidate `g.attr` calls